casbin/enforcer.py

```diff
@@ -191,6 +191,45 @@ def get_implicit_permissions_for_user(
 
         return res
 
+    def get_implicit_permissions_for_user_by_named_policy(
+        self, ptype, user, domain="", filter_policy_dom=True
```

Comment:
@Nekotoxin The name here is changed to `get_named_implicit_permissions_for_user`, will it be better?
@hsluoyz  what do you think?

Reply:
@leeqvip `get_named_implicit_permissions_for_user()` is great!

Reply:
have done it

Reply:
![image](https://user-images.githubusercontent.com/75596353/166089306-f76c4f23-200b-40ba-a545-624f41b21494.png)
![image](https://user-images.githubusercontent.com/75596353/166089317-5952e16d-8292-4311-b391-15361528a9aa.png)